wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearning/ActiveLearningKeyboardShortcuts.vue

```diff
@@ -55,6 +79,46 @@ export default Vue.extend({
                     previousCard();
                     break;
             }
+        },
+        editKeydownListener(event) {
+            // Using keyboard to set hotkeys
+            if (event.key === 'Enter') {
+                // The user has finalized their hotkey selection
+                const newKey = this.currentInput.join('+');
+                const hotKey = this.hotkeyFromIndex(this.currentlyEditing);
+                assignHotkey(hotKey, newKey);
+                this.resetEditingValues();
+                return;
+            }
+            this.currentInput = this.parseUserHotkeys(event);
+        },
+        keydownListener(event) {
+            event.preventDefault();
```

Comment:
This disables all keyboard interaction.  Can we just prevent the default when in edit mode or when the key is one of the hot keys?  Otherwise it breaks other browser interactions (like tab).

Reply:
Ah, yes, sorry about that! A bit too aggressive. This has been fixed now :+1: